libs/state-machine/src/lib/handleWizardRequest.ts

```diff
@@ -50,7 +50,7 @@ export async function handleWizardRequestAndReturnWizardResponse<
 						wizardLayout,
 						genericInterface,
 				  )
-				: await setupInitialState(requestBody, genericInterface);
+				: await setupInitialState(requestBody, wizardLayout, genericInterface);
```

Comment:
not really part of this PR but perhaps there is a more descriptive name for `genericInterface`. `genericStorageInterface` maybe - something that clarifies its function.

Reply:
sounds good - the naming does need to be addressed. We refer to this as a `storageInstance` in places, but potentially this interface won't necessary handle storage (or might handle storage but also something else) - I'll add an issue to look at it.